### ISSUE TITLE
Add phone number generator

### ISSUE THREAD

**Issue by lucapette** [lucapette/fakedata#79]
https://en.wikipedia.org/wiki/E.164

**Comment by ShivamJoker** [comment]
I also want this :)

**Comment by lucapette** [comment]
@ShivamJoker good to hear! Makes me want to build it faster :)

I didn't get to this yet as I'm not sure too sure about the UI of the command. What I thought we'd do is something like:


```sh
$ fakedata phone
# generates international numbers from all countries 
$ fakedata phone.[two digit country code]
# generates numbers for a specific country
```

Did you have something else in mind?

**Comment by ShivamJoker** [comment]
This looks good but it would be great if we could also have a  phone number without the country code, something like

```sh
fakedata phone.local[no. of digits]
```

I am not sure how it will work though we can have some arguments like `--no-seprator`

**Comment by lucapette** [comment]
ah! Nice :) I didn't think about the `fakedata phone.local` option. To control the number of digits, we'd use "constraints" like we do for [int](https://github.com/lucapette/fakedata/#int) for example.

So it would look like this:


```sh
$ fakedata phone.local:9
# 9-digit numbers
```

I don't _love_ the syntax (nor the naming "constraints", probably a simple "options" would do) but it's a little tricky to come up with something better (taking into account generators can repeat makes it tricky). I have some notes for a "fakedata 2.0" issues but didn't get around to share it yet. Soon!

**Comment by lucapette** [comment]
released in 1.5 🎉
